src/helpers/routes/routes.js

```diff
@@ -13,6 +13,15 @@ class APIRoutes {
   static veteransSignUpPath() {
     return APIRoutes.create('/veterans');
   }
+  static resourcePath() {
+    return APIRoutes.create('/resources');
+  }
+  static newUpvote() {
+    return APIRoutes.create('/upvotes');
+  }
+  static deleteUpvote(resourceId, veteranId) {
+    return APIRoutes.create(`/upvotes/delete_upvote/${resourceId}/${veteranId}`);
+  }
```

Comment:
This URL pattern probably needs some changing. In REST we expect the resource to be deleted on a call to DELETE /upvotes/id so if we want to use that route then we’d need to serialize the upvote id for each resource object. Otherwise we can think of another route like PATCH /upvotes/unupvote and pass in the resource and veteran id as params